scripts/homestead.rb

```diff
@@ -40,7 +40,7 @@ def Homestead.configure(config, settings)
     # Configure A Few VMware Settings
     ["vmware_fusion", "vmware_workstation"].each do |vmware|
       config.vm.provider vmware do |v|
-        v.vmx["displayName"] = "homestead"
+        v.vmx["displayName"] = settings["name"] ||= "homestead-7"
```

Comment:
I just added the same logic from virtual box provider:

https://github.com/laravel/homestead/blob/master/scripts/homestead.rb#L32